react/country/CHL.js

```diff
@@ -506,6 +510,27 @@ export default {
       valueIn: 'long_name',
       types: ['postal_code'],
       required: false,
+      handler: address => {
+        if (
+          !address.state ||
+          !address.state.value ||
+          !address.neighborhood ||
+          !address.neighborhood.value
+        ) {
+          return address
+        }
+
+        if (
+          countryData[address.state.value] &&
+          countryData[address.state.value][address.neighborhood.value]
+        ) {
+          address.postalCode = {
+            value: countryData[address.state.value][address.neighborhood.value],
+          }
+        }
```

Comment:
I think you can do something like this:
```suggestion
        address.postalCode = {
          value: countryData.[address.state.value]?.[address.neighborhood.value],
        }
```
I didn't test this code, though

Reply:
Not really, the optional chaining is fine but I shouldn't touch the field if I don't have a postal code for it.

Reply:
It makes sense. So, you could use the optional chaining in your if-statement instead